crates/teloxide-core/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## unreleased
 
+## 0.10.0 - 2023-02-15
```

Comment:
I think we've missed `ApiError::ImageProcessFailed` in the changelog from https://github.com/teloxide/teloxide/pull/825.